numba/future.py

```diff
@@ -0,0 +1,2 @@
+def disable_reflected_list():
```

Comment:
Better to add docstring to this function

Reply:
and also docstring for the file

Reply:
Added, please review.

Reply:
ping @sklam